src/core/middleware/logRequests.js

```diff
@@ -7,5 +7,5 @@ export function logRequests(req, res, next) {
   next();
   const finish = new Date();
   const elapsed = finish - start;
-  log.info({ req, res, start, finish, elapsed });
+  log.debug({ req, res, start, finish, elapsed });
```

Comment:
🌈 

When you actually turn on debug, though, the server log is pretty much useless. Ha. I am really not sure about this middleware.